dCommon/Diagnostics.cpp

```diff
@@ -197,17 +197,6 @@ void MakeBacktrace() {
 	sigact.sa_sigaction = CritErrHdlr;
 	sigact.sa_flags = SA_RESTART | SA_SIGINFO;
 
-	if (sigaction(SIGSEGV, &sigact, (struct sigaction*)nullptr) != 0 ||
-		sigaction(SIGFPE, &sigact, (struct sigaction*)nullptr) != 0 ||
-		sigaction(SIGABRT, &sigact, (struct sigaction*)nullptr) != 0 ||
-		sigaction(SIGILL, &sigact, (struct sigaction*)nullptr) != 0) {
-		fprintf(stderr, "error setting signal handler for %d (%s)\n",
-			SIGSEGV,
-			strsignal(SIGSEGV));
-
-		exit(EXIT_FAILURE);
-	}
-
```

Comment:
Why is this removed?

Reply:
Didn't you tell me to remove it?

Reply:
https://github.com/DarkflameUniverse/DarkflameServer/pull/1361#discussion_r1436625455

Reply:
Oh I meant the casts.  It's nullptr, it will always be cast to the correct ptr type